UI/src/utils/styleRule.js

```diff
@@ -110,6 +110,16 @@
             return this;
         };
 
+      this.isBetween = function(values){
+        var expression = expressionFn[expressionFn.length-1];
```

Comment:
Probably could be checked if values is an array and if it has max/min lenght = 2

Reply:
Done